temporal-sdk/src/main/java/io/temporal/client/WorkflowOptions.java

```diff
@@ -374,10 +374,8 @@ public Builder setDisableEagerExecution(boolean disableEagerExecution) {
     }
 
     /**
-     * Time to wait before dispatching the first workflow task. If the workflow gets a signal before
-     * the delay, a workflow task will be dispatched and the rest of the delay will be ignored. A
-     * signal from signal with start will not trigger a workflow task. Cannot be set the same time
-     * as a CronSchedule.
+     * Time to wait before dispatching the first workflow task. A signal from signal with start will
+     * not trigger a workflow task. Cannot be set the same time as a CronSchedule.
```

Comment:
```suggestion
     * Time to wait before dispatching the first workflow task. A signal included in signal with start will
     * not trigger a workflow task. Cannot be set the same time as a CronSchedule.
```

Reply:
Going to not take, only because I think we will be removing this whole line about signal with start